.nvmrc

```diff
@@ -0,0 +1 @@
+v8.12.0
```

Comment:
Hey @JavonDavis , can you expand on the need to have this configuration setting?

Reply:
This was just a convenience since this is the node versio recommended in the README for those who use the nvmrc file to automatically switch environments. I also think it was also auto added my IntelliJ and I didn't notice, there's no problem to remove it if there's an issue.

Reply:
It'd be weird to have a hard version pinning for the tests and not for the rest of the project so, it's better to either remove this (simpler) or expand it to the whole app project (enlarges the scope though).

I think removing it might be preferable, but no strong feelings.